instrumentation/runtime-metrics/library/README.md

```diff
@@ -0,0 +1,80 @@
+# JVM Runtime Metrics
+
+This module provides JVM runtime metrics as documented in the [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#jvm-metrics).
+
+## Quickstart
+
+### Add these dependencies to your project:
+
+Replace `OPENTELEMETRY_VERSION` with the [latest
+release](https://search.maven.org/search?q=g:io.opentelemetry.instrumentation%20AND%20a:opentelemetry-runtime-metrics).
+
+For Maven, add to your `pom.xml` dependencies:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-runtime-metrics</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+</dependencies>
+```
+
+For Gradle, add to your dependencies:
+
+```groovy
+runtimeOnly("io.opentelemetry.instrumentation:opentelemetry-runtime-metrics:OPENTELEMETRY_VERSION")
+```
+
+### Usage
+
+Register observers for the desired runtime metrics:
+
+```java
+OpenTelemetry opentelemetry = // OpenTelemetry instance configured elsewhere
+
+BufferPools.registerObservers(opentelemetry);
+Classes.registerObservers(opentelemetry);
+Cpu.registerObservers(opentelemetry);
+MemoryPools.registerObservers(opentelemetry);
+Threads.registerObservers(opentelemetry);
+GarbageCollector.registerObservers(opentelemetry);
+```
+
+## Garbage Collector Dependent Metrics
```

Comment:
maybe just(?)
```suggestion
## Garbage Collector Metrics
```

Reply:
Well the memory metrics are dependent on the garbage collector as well, since the gc dictates the organization of memory pools